src/features/user/Settings/EditProfile/EditProfileForm.tsx

```diff
@@ -287,32 +307,40 @@ export default function EditProfile({}: Props) {
             </div>
           </label>
         </div>
+
         {type !== 'tpo' ? (
-          <MaterialTextField
-            label={t('editProfile:iamA')}
-            variant="outlined"
-            select
-            defaultValue={type}
-          >
-            {profileTypes.map((option) => (
-              <MenuItem
-                key={option.value}
-                value={option.value}
-                onClick={() => setAccountType(option.value)}
-              >
+          <MuiAutoComplete
```

Comment:
You could also use <ReactHookFormSelect> over here, as used in BankDetailsForm / PayoutScheduleForm, if you want to make this available to ReactHookForm.

Reply:
Left as is as this needs a greater refactor to the code of the `ReactHookFormSelect` component to make it more generic.